dGame/dComponents/PropertyEntranceComponent.h

```diff
@@ -59,15 +61,17 @@ class PropertyEntranceComponent : public Component {
 	 */
 	[[nodiscard]] LWOMAPID GetMapID() const { return m_MapID; };
 
-	PropertySelectQueryProperty SetPropertyValues(PropertySelectQueryProperty property, LWOCLONEID cloneId = LWOCLONEID_INVALID, std::string ownerName = "", std::string propertyName = "", std::string propertyDescription = "", float reputation = 0, bool isBFF = false, bool isFriend = false, bool isModeratorApproved = false, bool isAlt = false, bool isOwned = false, uint32_t privacyOption = 0, uint32_t timeLastUpdated = 0, float performanceCost = 0.0f);
-
-	std::string BuildQuery(Entity* entity, int32_t sortMethod, Character* character, std::string customQuery = "", bool wantLimits = true);
+	void PopulateUserFriendMap(uint32_t user);
 
+	std::vector<uint32_t> GetPropertyIDsBasedOnParams(const std::string& searchText, uint32_t sortMethod, Entity* requestor);
```

Comment:
BasedOnParams seems a bit too explicit here and I dont believe it helps tell a reader what the function does.  We could probably just remove the BasedOnParams and describe in a comment block how the function arguments affect the lookup